### PR TITLE
Fixed "WebAssembly Filters" link in the footer

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: YashKamboj <75125203+YashKamboj@users.noreply.github.com>

**Description**
Fixed "WebAssembly Filters" link in the footer
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/General/Footer/index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const Footer = () => {
                   <li>
                     <Link
                       className="category-link"
-                      to="/resources/category/webassembly-filters"
+                      to="/learn/learning-paths"
                     >WebAssembly Filters
```
Comment (Abhijay007) [inline src/sections/General/Footer/index.js:51]: replace text "WebAssembly Filters" to "Learning Paths"

**Review by Abhijay007 — CHANGES_REQUESTED** [review]

File: src/sections/General/Footer/index.js (hunk 1 above)
Comment (warunicorn19) [reply src/sections/General/Footer/index.js:51]: Lol, good catch.😂 I really didnt see it.

Comment (YashKamboj) [reply src/sections/General/Footer/index.js:51]: https://layer5io.slack.com/archives/C015QJKUMPU/p1642531158024400 @Abhijay007

Comment (warunicorn19) [reply src/sections/General/Footer/index.js:51]: @YashKamboj the text also needs to be replaced.

Comment (warunicorn19) [inline src/sections/General/Footer/index.js:51]:
```suggestion
                    >WebAssembly Filters
```

**Pull request merged**